docs/new_labels.md

```diff
@@ -0,0 +1,25 @@
+# Adding New Labels
+New Docker Labels may be created and/or updated for the output image by creating a new label file.
+
+If a new label is specified in the file, the label will simply be added in the output image.  However, any label previosly defined in the base builder image will be ***overwritten*** in the output image, if the same label name is specified in the labels file.
+
+## File Name and Path
+The name and path of the labels file ***must*** be the following:
+```bash
+/opt/docker/new_labels.json
+```
+
+## Example 
+The file may have 1 or more label/value pair.  Below is the JSON format of the new labels file:
```

Comment:
thank you.  I will need to almost redo this doc since I've only focused on adding labels.